straxen/plugins/nveto_recorder.py

```diff
@@ -41,7 +42,7 @@ class nVETORecorder(strax.Plugin):
 
     rechunk_on_save = True
     save_when = strax.SaveWhen.TARGET
-    compressor = 'lz4'
+    compressor = 'zstd'
```

Comment:
Thanks for the change Daniel, one think to keep in mind is that zstd has a max input buffer of 2 GB. If we ever hit 2000 MB/chunk = 400 MB/s we might get some issues here. However, I think at that stage we have a great deal of other issues too, it's also orders of magnitude higher than we'd like to be saving anyway.

Might be something to keep in mind for NG calibration. We could also of course decrease the chunk size.

At this stage, you haven't rechunked the data yet but given the coincidence requirements we should be reducing the datarate already by a lot.

Reply:
Thanks, this I did not knew, but I also do not quite understand this. I though we would cap our maximum chunksize? So how can we ever hit 2000MB/chunk, or is the 400MB/s the important hard limit to remember here? If this is the case, the nVETO DAQ can not record more then ~60ish to 70ish MB/s so we should never hit this problem here.

Reply:
The problem is the 2000MB/chunk, if it's bigger than this the compressor will simply die. 

My point is exactly this:
>I though we ~would~ SHOULD cap our maximum chunksize?

I agree on this, was just saying that we should not forget although I also cannot imagine we will run into this problem
> we should never hit this problem here.

To replicate just make a 2.5 GB array and try to save it using zstd or blosc using the save function from strax.io.